Adding calendar events for next 7 days

samworley opened the pull request:
Have updated script to add events for the next 7 days, ensuring to remove existing events so as not to produce duplicates. This means the script can be run multiple times without worrying, as no multiples are created.

Have also removed deprecated file from README and fixed check of calendarsWithName length.

samworley committed:
Delete gSuite_to_personal.txt

samworley committed:
Update README.md

Removed deprecated part, updated to include 7 day change

samworley committed:
Update and rename google_home_mutiple_calendars.txt to google_home_mutiple_calendars.js

Fixed check on size of name object, added event description, delete existing events, add events for next 7 days

samworley committed:
Update README.md

prashantkhurana commented:
Thank you! This was much needed and I have wanted to do something similar for some time now, so thanks for getting this done. I will be setting up a google mini from scratch soon, so I'll be able to test it out. Give me a few days, but reading the .js, this looks perfect!

samworley commented:
No problem! I found your post on Reddit when I was having issues last night and your script was really helpful.
There are probably some other things that could be changed, but let me know if you need anything :)

samworley committed:
Update google_home_mutiple_calendars.js

-- fixed issue where only first event of each day was used
-- now starts on current day first
-- removed space at beginning of event name if only one calendar in use

samworley commented:
Just fixed a couple of issues - only the first event from each day was being used and now it includes the current day as well, as this was causing a couple of problems for me.

prashantkhurana commented:
Sorry for the delay, finally got to take a look at it. Seems like this would delete all the events of the google home calendar before copying the events over. That would mean that any event which is only present in google_home_calendar will be deleted. I think, something clever needs to be done to only delete events which are from the calendar we will copy later on. Seems like we can use [tags](https://developers.google.com/apps-script/reference/calendar/calendar-event#setTag(String,String)) to find out which events can we delete. I can do that and push to that change to this PR, if you don't mind.

samworley commented:
Yeah that's true, I hadn't really considered that as I only use it to display events from another calendar - but feel free to make changes!
